src/hooks.ts

```diff
@@ -0,0 +1,25 @@
+import { getInitialLocale } from '$lib/utils/i18n/initialization';
+import { addLocaleToResponseBody, addLocaleRelativeRoutePath } from '$lib/utils/i18n/modifier';
+
+/** @type {import('@sveltejs/kit').GetContext} */
+export async function getContext(request) {
+	return {
+		locale: getInitialLocale(request)
```

Comment:
This lets us get the locale and pass it to the session where it can be used like:
https://github.com/kobejean/svelte-kit-i18n/blob/e32df541b2388d740a8109f4a7270129f3c8ef36/src/routes/%24layout.svelte#L38-L40